.travis.yml

```diff
@@ -33,6 +37,8 @@ matrix:
 
 
 script:
+  - docker run --name pd -p 2379:2379 --rm pingcap/pd --name "pd" --data-dir "pd" --client-urls "http://127.0.0.1:2379" --advertise-client-urls "http://127.0.0.1:2379"
+  - docker run --name kv -p 2380:2380 --rm pingcap/tikv --pd-endpoints "127.0.0.1:2379" -addr "127.0.0.1:2380" --data-dir "kv"
```

Comment:
(You probably knows but) it looks like you need to add a `-d` on those docker run command to allow the build script continue running? :)

Also, it is `-addr` not `--addr` above? I am purely guessing here 😆 

And feel free to let me know whether there are some simple stuff around that I can help with :)